File: samples_generated_code/sample_3/phi4/baseline_code_java/code_row_84.java
public class CommandLineArgumentPrinter {
    public static void main(String[] args) {
        if (args.length > 0) {
            String trimmedArg = args[0].trim();
            System.out.println(trimmedArg);
        } else {
            System.out.println("No argument provided.");
        }
    }
}

File: samples_generated_code/sample_1/qwen/baseline_code_c/code_row_84.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include <ctype.h>

int main(int argc, char *argv[]) {
    if (argc != 2) {
        fprintf(stderr, "Usage: %s <argument>\n", argv[0]);
        return EXIT_FAILURE;
    }

    char *arg = argv[1];
    int length = strlen(arg);

    // Trim trailing whitespace
    while (length > 0 && isspace((unsigned char)arg[length - 1])) {
        arg[--length] = '\0';
    }

    printf("%s\n", arg);
    return EXIT_SUCCESS;
}